py/desitarget/sv1/sv1_cuts.py

```diff
@@ -1506,19 +1499,21 @@ def set_target_bits(photsys_north, photsys_south, obs_rflux,
             lrg_classes[int(south)] = isLRG(
                 primary=primary, south=south,
                 gflux=gflux, rflux=rflux, zflux=zflux, w1flux=w1flux,
-                zfiberflux=zfiberflux, rflux_snr=rsnr, zflux_snr=zsnr,
-                w1flux_snr=w1snr, gnobs=gnobs, rnobs=rnobs, znobs=znobs,
+                zfiberflux=zfiberflux, rfluxivar=rfluxivar, zfluxivar=zfluxivar,
+                w1fluxivar=w1fluxivar, gnobs=gnobs, rnobs=rnobs, znobs=znobs,
                 maskbits=maskbits
             )
-    lrg_north, lrginit_n_4, lrgsup_n_4, lrginit_n_8, lrgsup_n_8 = lrg_classes[0]
-    lrg_south, lrginit_s_4, lrgsup_s_4, lrginit_s_8, lrgsup_s_8 = lrg_classes[1]
+    lrg_opt_n, lrg_ir_n, lrg_sv_opt_n, lrg_sv_ir_n = lrg_classes[0]
```

Comment:
About a dozen lines above this line of code, you write:
```
    lrg_classes = [[tcfalse, tcfalse, tcfalse, tcfalse, tcfalse],
                          [tcfalse, tcfalse, tcfalse, tcfalse, tcfalse]]
```
But, you've changed to only _four_ selections. So, you need to update the defaults in that line of code to:
```
    lrg_classes = [[tcfalse, tcfalse, tcfalse, tcfalse],
                          [tcfalse, tcfalse, tcfalse, tcfalse]]
```
This is triggered in the event that someone passes a `--tcnames` that does not include the LRGs.

Reply:
Thanks. It's fixed.